api-flask/app/main.py

```diff
@@ -92,13 +99,32 @@ def stats():
             url=zones_url
         )
 
+    wfs_params = data.get('wfs_params', None)
+
+    wfs_response = None
```

Comment:
Let's add a bit more details here in a comment and update the README as well with the new input parameter and its behavior.

Ideally, add some quick tests if it's not too complicated

Then I think we should be good to go :)

Reply:
included in https://github.com/WFP-VAM/prism-frontend/pull/186/commits/2f0aa45006bc31f0c60d246aff55f67f4b49f584 and https://github.com/WFP-VAM/prism-frontend/pull/186/commits/286645f27c106a2ca4813612fd135fc77791bfd2